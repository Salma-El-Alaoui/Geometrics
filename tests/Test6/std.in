C c1 1 1 1
LIST
DELETE c1
LIST
R r1 1 1 1 1
L l1 1 1 1 1
LIST
DELETE r1
LIST
DELETE l1
LIST
EXIT
