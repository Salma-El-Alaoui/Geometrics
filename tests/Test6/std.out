OK
#Element ajoute: c1
C c1 1 1 1
#Element c1 supprime
OK
OK
#Element ajoute: r1
OK
#Element ajoute: l1
L l1 1 1 1 1
R r1 1 1 1 1
#Element r1 supprime
OK
L l1 1 1 1 1
#Element l1 supprime
OK
